File: 所有课程/java/thinking of java/unit7/src/localhost/test2.java
package localhost;
public class test2 {
	static public void main(String[] args){
		class A{
			private String i = "123";
			protected void add(String s){i+=s;};
			protected void print(){System.out.print(i);}
		}
		class B extends A{
			
			public void add(int s){
				System.out.print("add");
				
				super.i += s;
			}
			B(){
				/*super.add("123");
				super.add("321");
				super.add("123");*/
				add(1);
				
				super.print();
			}
		}
		new B();
	}
}

File: Deno/API/Fetch/test2.js
async function sendRequest() {
    const url = "http://localhost:3000/shop-service/v1/app";
    try {
        const response = await fetch(url);
        if (response.ok) {
            const data = await response.json();
            console.log("Response:", data);
        }
        else {
            console.log("Request failed with status:", response.status);
        }
    }
    catch (error) {
        console.log("Error:", error.message);
    }
}
sendRequest();
/*
deno run --allow-net test2.ts
*/ 
